lib/std/fs/path.zig

```diff
@@ -468,6 +468,9 @@ fn asciiEqlIgnoreCase(s1: []const u8, s2: []const u8) bool {
 }
 
 /// On Windows, this calls `resolveWindows` and on POSIX it calls `resolvePosix`.
+/// Note: Dont use for symlinks, because Windows and Posix having different behavior.
```

Comment:
This is misleading - `resolve` currently makes no attempt to resolve symlinks on any platform

Reply:
added
```txt
/// Note: Symlinks currently dont work and should not be used,
/// because Windows and Posix having different behavior.
```